pkg/m3admin/placement/types.go

```diff
@@ -38,4 +38,6 @@ type Client interface {
 	Add(instance placementpb.Instance) error
 	// Remove removes a given instance with the given ID from the placement.
 	Remove(id string) error
+	// Replace replaces one instance with another.
+	Replace(leavingID string, newInstance placementpb.Instance) error
```

Comment:
i don't know how much i like the name `leavingID` but i don't have any better ideas lol

Reply:
Ah ya know what, I'm going to change it to `leavingInstanceID` to match the placement service [API](https://godoc.org/github.com/m3db/m3cluster/placement#Service) and the [proto](https://github.com/m3db/m3/blob/85841a3ff2b11b69549e6ae2f19cfa750408c850/src/query/generated/proto/admin/placement.proto#L25)